module5/projects/CallingAnApiMiniProject.md

```diff
@@ -0,0 +1,54 @@
+---
+layout: page
+title: Calling An API Mini Project
+---
+
+## Learning Goals
+* Practice the skill of calling an API from a .NET application
+* Learn how to make a PUT and DELETE request to an API
+* Gain experience navigating vague directions for how to approach a problem
+
+## Overview
+
+For this mini project, imagine that you are working on a team with other developers. One of your teammate is responsible for building an API for keeping track of someone's favorite gifs. Your job is to build a .NET MVC application to call your teammate's API and allow for user interaction.
+
+For this project you will be working in pairs and utilizing driver/navigator pair programming.
+
+A new tool you'll run into with this project is [Swagger](https://swagger.io/)! Let's take a detour to [this](../lessons/Week4/Swagger) mini lesson on Swagger.
+
+## The API
+
+[This](https://github.com/turingschool-examples/GifTracker) is the API repo your teammate is working on. Clone down this repo and checkout the branch `GET-all-gifs`. When you start up the application you should be directed to a Swagger page.
+
+## The User Experience - Your Job!
+
+The pair of you are responsible for creating a separate application that calls the API to get all gifs, create a gif, update a gif, and delete a gif. How you build the UI for these features is up to you! You're welcome to either use the tools we have covered in class or dive more into javascript.

```

Comment:
This might be more clear if we say something like: "calls the API endpoints implemented in GifTracker, so that users of your application can: get all gifs, create a gif..."

I'm thinking there will probably be some initial confusion about what we are even expecting them to do 🙃